workspaces/marketplace/examples/plugins/tekton.yaml

```diff
@@ -243,8 +243,6 @@ spec:
     ![tekton-pipelinerun-report](https://github.com/backstage/community-plugins/raw/main/workspaces/tekton/plugins/tekton/docs/images/tekton-plugin-pipelinerun-output.png)
 
   installation: |
-    # Setting up the Tekton plugin
-
```

Comment:
Please revert this. We should not assume specific markdowns. We can talk about that later.